atd-etl/cris_import/build_vz_contents.py

```diff
@@ -0,0 +1,559 @@
+            "opvault": VAULT_ID,
+            },
+    }
+
+    # instantiate a 1Password client
+    client: Client = new_client(ONEPASSWORD_CONNECT_HOST, ONEPASSWORD_CONNECT_TOKEN)
+    # get the requested secrets from 1Password
+    return onepasswordconnectsdk.load_dict(client, REQUIRED_SECRETS)
+
+
+def get_pg_connection():
+    """
+    Returns a connection to the Postgres database
+    """
+    return psycopg2.connect(
+        host=DB_HOST,
+        user=DB_USER,
+        password=DB_PASS,
+        dbname=DB_NAME,
+        sslmode=DB_SSL_REQUIREMENT,
+        sslrootcert="/root/rds-combined-ca-bundle.pem",
+    )
+
+
+def compute_for_crashes():
+    with SshKeyTempDir() as key_directory:
+        write_key_to_file(key_directory + "/id_ed25519", DB_BASTION_HOST_SSH_PRIVATE_KEY + "\n") 
+        ssh_tunnel = SSHTunnelForwarder(
+            (DB_BASTION_HOST),
+            ssh_username=DB_BASTION_HOST_SSH_USERNAME,
+            ssh_private_key=f"{key_directory}/id_ed25519",
+            remote_bind_address=(DB_RDS_HOST, 5432),
+        )
+        ssh_tunnel.start()
+
+        pg = psycopg2.connect(
+            host="localhost",
+            port=ssh_tunnel.local_bind_port,
+            user=DB_USER,
+            password=DB_PASS,
+            dbname=DB_NAME,
+            sslmode=DB_SSL_REQUIREMENT,
+            sslrootcert="/root/rds-combined-ca-bundle.pem",
+        )
+
+        cris_cursor = pg.cursor(cursor_factory=psycopg2.extras.RealDictCursor)
+        public_cursor = pg.cursor(cursor_factory=psycopg2.extras.RealDictCursor)
+        vz_cursor = pg.cursor(cursor_factory=psycopg2.extras.RealDictCursor)
+
+        vz_cursor.execute('truncate vz_facts.atd_txdot_crashes cascade')
+        pg.commit()
+
+        sql = """
+            select * from cris_facts.atd_txdot_crashes 
+            order by crash_id asc
+            """
+        cris_cursor.execute(sql)
+        for cris in cris_cursor:
+
+            print()
+            print("CRIS: ", cris["crash_id"])
+            sql = "select * from production_fact_tables.atd_txdot_crashes where crash_id = %s"
+            public_cursor.execute(sql, (int(cris["crash_id"]),))
+            public = public_cursor.fetchone()
+            #print("public: ", public)
+            sql = None
+            keys = ()
+            values = ()
+            if public is None:
+                # we have a crash in CRIS that is not in the old VZDB
+                #sql = "insert into vz_facts.atd_txdot_crashes (crash_id) values (%s)"
+                keys = ["crash_id"]
+                values = [cris["crash_id"]]
+                print("Crash missing from old VZ data: ", cris["crash_id"])
+            else:
+                # print("public: ", public["crash_id"])
+                keys = ["crash_id"]
+                values = [cris["crash_id"]]
+                for k, v in cris.items():
+                    if (k in ('crash_id', 'council_district', 'in_austin_full_purpose')): # use to define fields to ignore
```

Comment:
We spoke about this today, and I don't know if we came to a full resolution on what we should do. I think we should go the way that makes writing the triggers easier, and if all else is equal, I think we evaluate them on a case-by-case basis. 

My gut is to get this PR in, even if it's handling almost all fields "generically" and we can create an issue that is limited in scope to audit and choose exactly how we're going to handle each field non-trivial field. This PR can be a scaffold and starting place for us to do that work. Essentially, I want to avoid packing as much LDM work as we can into this already huge PR, and start to break out refinements to the system systematically and intentionally. 

@patrickm02L - I want to tag you since I'm proposing a planning-like operation, and I want to make sure you're 👍 or give you a chance to offer direction if you want.

Reply:
Cool, I'm going to get one written up today and put it in the epic. Thanks!

Reply:
Here's an issue describing this work to refine the special handling of columns in the unifying view.

https://github.com/cityofaustin/atd-data-tech/issues/14008